modules/flowable-ui/flowable-ui-modeler-frontend/src/main/resources/static/modeler/views/popup/process-import.html

```diff
@@ -65,14 +72,18 @@ <h2>{{'PROCESS.POPUP.IMPORT-TITLE' | translate}}</h2>
                 </div>
 
                 <!-- <![endif]-->
+
+                <div class="form-check">
+                    <input class="form-check-input" type="checkbox" ng-model="model.importNewVersionIndicator" id="importNewVersionIndicator">
+                    <label class="form-check-label" for="importNewVersionIndicator">
+                        Import into existing process model as new version if keys are matching.
```

Comment:
Shouldn't this be something like:
````
 {{'PROCESS.POPUP.IMPORT-NEW-VERSION' | translate}}
````
and then the string added to `modules/flowable-ui/flowable-ui-modeler-frontend/src/main/resources/static/modeler/i18n/en.json` some where around line 2153 so it can be localized?